src/RokuAdapter.ts

```diff
@@ -1144,7 +1196,7 @@ export class RequestPipeline {
     }
 
     /**
-     * Internall request processing function
+     * Internally request processing function
```

Comment:
I think the word should be `Internal`. Spell checker's getting a little aggressive. :P